convert2rhel/actions/system_checks/convert2rhel_latest.py

```diff
@@ -91,6 +97,15 @@ def run(self):
                 "Couldn't check if the current installed Convert2RHEL is the latest version.\n"
                 "repoquery failed with the following output:\n%s" % (raw_output_convert2rhel_versions)
             )
+            self.add_message(
+                level="WARNING",
+                id="CONVERT2RHEL_LATEST_CHECK_SKIP",
+                title="Skipping convert2rhel latest version check",
+                description=(
+                    "Couldn't check if the current installed Convert2RHEL is the latest version.\n"
+                    "repoquery failed with the following output:\n%s" % (raw_output_convert2rhel_versions)
+                ),
```

Comment:
Maybe this description should be moved to a variable to not mismatch at some point from the warning logger?